pkgs/applications/audio/littlegptracker/default.nix

```
@@ -1,75 +1,63 @@
 { lib, stdenv
 , fetchFromGitHub
-, unstableGitUpdater
-, SDL
+, SDL2
+, python3
 , jack2
 , Foundation
+, alsa-lib
+, pkg-config
 }:
-
-stdenv.mkDerivation {
+let
+  python = python3.withPackages (ps: with ps; [ pillow ]);
+  platform = if stdenv.hostPlatform.isLinux then "X64" else "OSX";
+in stdenv.mkDerivation rec {
   pname = "littlegptracker";
-  version = "0-unstable-2020-11-26";
+  version = "1.4.2";
 
   src = fetchFromGitHub {
-    owner = "Mdashdotdashn";
+    owner = "djdiskmachine";
     repo = "littlegptracker";
-    rev = "4aca8cd765e1ad586da62decd019e66cb64b45b8";
-    sha256 = "0f2ip8z5wxk8fvlw47mczsbcrzh4nh1hgw1fwf5gjrqnzm8v111x";
+    rev = "refs/tags/${version}";
+    hash = "sha256-1uXC5nJ63YguQuNIkuK0yx9lmrMBqw0WdlmCV8o11cE=";
   };
 
   buildInputs = [
-    SDL
+    pkg-config
+    SDL2
+    python
+    alsa-lib
   ]
   ++ lib.optional stdenv.hostPlatform.isDarwin Foundation
   ++ lib.optional stdenv.hostPlatform.isLinux jack2;
 
-  patches = [
-    # Remove outdated (pre-64bit) checks that would fail on modern platforms
-    # (see description in patch file)
-    ./0001-Remove-coherency-checks.patch
-  ];
-
   preBuild = "cd projects";
```

Let's format this hook properly while we're at it

```suggestion
  preBuild = ''
    cd projects
  '';
```